setup/tinyHIPPOstart

```diff
@@ -8,11 +8,13 @@
 
 # Update the package repositories for OpenWrt
 opkg update
+opkg install nmap
```

Comment:
Please remove this opkg install. The OpenWrt makefile takes care of any OpenWrt packages that need to be installed